src/granite/select.cr

```diff
@@ -0,0 +1,21 @@
+module Granite::Select
+  struct Container
+    property custom
+    getter table_name, fields
```

Comment:
Since you’re target is to support multiple tables, shouldn’t each field have a paired table?

Reply:
@robacarp there may be no tables or fields that are calculated.  When this is a custom query, the table_name will not have any meaning.

What I think may be a bigger issue is all of the transactional methods are still included when a custom query is provided.  These methods should be excluded like `save` and `destroy` or at least throw runtime exceptions if you attempt to use them.  But it would be better to catch these at compile time.

Reply:
@drujensen Agreed.  I think the next step (and I'll write up an issue to discuss this) is to go with your original suggestion in #184 and fully break this out into a ViewModel, which will eliminate the transactional methods (and, hence, no need for compiler check).